### DISCUSSION TITLE
Authentication failed (error 3 and 8)

### DISCUSSION THREAD

**Discussion by IljaI** [community/community#22759] [New to GitHub]
<p>I tried cloning project from gitlab to github desktop for the first time. After short loading it asks for username and password, and I am not sure if it want account for gitlab or github. No matter how many times I try to enter my data from both github and gitlab it appears again. Even creating new account on gitlab and using it’s username and password did not worked.<br>
Log give those two warning during process :</p>
<pre><code>2018-05-04T18:11:03.481Z - warn: [ui] fetchRepository: 'Denis_hroft/A.S.H.' returned a 404
2018-05-04T18:11:03.629Z - warn: [ui] fetchRepository: 'Denis_hroft/A.S.H.' returned a 404
</code></pre>
<p>And ends with this :</p>
<pre><code>2018-05-04T18:11:05.278Z - error: [ui] `git -c credential.helper= clone --recursive --progress -- https://gitlab.com/Denis_hroft/A.S.H..git C:\Users\ilja\Desktop\ASH_gameProjects\UNREAL STUFF\GitLab UE project\A.S.H.` exited with an unexpected code: 128.
Cloning into 'C:\Users\ilja\Desktop\ASH_gameProjects\UNREAL STUFF\GitLab UE project\A.S.H.'...
remote: HTTP Basic: Access denied
fatal: Authentication failed for 'https://gitlab.com/Denis_hroft/A.S.H..git/'

(The error was parsed as 3: Authentication failed. You may not have permission to access the repository or the repository may have been archived. Open options and verify that you're signed in with an account that has permission to access this repository.)
</code></pre>
<p>Logs started sometimes showing this error, after my fuss with accounts today probably :</p>
<pre><code>exited with an unexpected code: 128.
Cloning into 'C:\Users\ilja\Desktop\ASH_gameProjects\UNREAL STUFF\GitLab UE project\A.S.H.'...
remote: The project you were looking for could not be found.
fatal: repository 'https://gitlab.com/Denis_hroft/A.S.H..git/' not found

(The error was parsed as 8: The repository does not seem to exist anymore. You may not have access, or it may have been deleted or renamed.)
</code></pre>
<p>Maybe my problem is trivial and I am missing some kind of easy and obvious step. But I just don’t know what to do anymore.<br>
<img src="https://user-images.githubusercontent.com/31922838/181073033-72b39294-567f-4238-a84f-b97eb955d810.png" alt="Shot.PNG" data-base62-sha1="2T7tf4566DHEIOlZ4t8Pj5p8ns4" width="475" height="332"></p>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/iljai">@iljai</a>,</p>
<p>Looking at the information that you provided, what I can tell you is that the credentials that GitHub Desktop is looking for in this case are your GitLab credentials. The error that you’re seeing  generally refers to when a git repository is not available to you either because the credentials you are using are wrong, your user account does not have access to the repository in question, or the account no longer exists at that address.</p>
<p>Thanks!</p>

**Comment by IljaI** [comment]
<p>Thank you for answering.<br>
What could be the causes of not having access to the repository? I know for sure that 2-FA is disabled and other people can access it normally.<br>
Also, <em>username</em> means, for example, just writing “IljaI” ?</p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Hi <a class="mention" href="/iljai">@iljai</a>,</p>
<p>It would probably be whatever your GitLab username is. However, since GitHub and GitLab are unaffiliated, I am not able to verify that for you.</p>
<p>Is this a public repository that you are trying to access?</p>
<p>Thanks!</p>

**Reply by himanshukoshti96** [reply]
yes public repository

**Reply by himanshukoshti96** [reply]
error 403 occured

**Comment by IljaI** [comment]
<p>I’m sorry, it turned out that project owner forgot that he should give access right to the people who will work with repository. Now I’m downloading content, and I think there will be no problems further on.<br>
Thanks for your concern and attention.</p>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/iljai">@iljai</a>,</p>
<p>You’re very welcome! I’m glad that you got that sorted.</p>
<p>Have a great day!</p>

**Comment by showravs** [comment]
<p>whenever i enter any of the links of  git clone it asks for username and password. how can i solve this?? i added a troublesome photo here</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/2/d/2da4d13ad16c56e82cb94a566e5d91416745a82d.jpeg" data-download-href="/uploads/short-url/6vMBxYZkVbjftZnwn9FCf1iHKRT.jpeg?dl=1" title="IMG_20201013_091954_932" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/72573901/181073040-e709292e-4373-4d07-bf62-112cafdee853.jpeg" alt="IMG_20201013_091954_932" data-base62-sha1="6vMBxYZkVbjftZnwn9FCf1iHKRT" width="294" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/2/d/2da4d13ad16c56e82cb94a566e5d91416745a82d_2_294x500.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/2/d/2da4d13ad16c56e82cb94a566e5d91416745a82d_2_441x750.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/3X/2/d/2da4d13ad16c56e82cb94a566e5d91416745a82d.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/d/2da4d13ad16c56e82cb94a566e5d91416745a82d_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">IMG_20201013_091954_932</span><span class="informations">480×816 79.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
